src/bidiMapper/domains/context/context.ts

```diff
@@ -68,6 +70,15 @@ export class Context {
     await this._cdpClient.Page.enable();
     await this._cdpClient.Page.setLifecycleEventsEnabled({ enabled: true });
     this._initializeEventListeners();
+
+    // initialize LogEntry
+    LogEntry.create(
```

Comment:
The naming confuses me a lot. I'd expect is to be something like `LogHandler` or `LogManager`. It's a bit different from the `Context`. `Context` is an instance representing the real context with a given `contextId`. Following the same logic, `LogEntry` should represent a single instance of `LogEntry`, while it represents a logic dealing with all the log events in the given context.

Reply:
We can either move all the `Log` domain logic into `LogManager` (like you did), or make the `LogEntry` a class representing a single instance with some formatting logic. The second approach would be way easier to test.

Basically, it's only the `format` logic made me ask you to move it to a dedicated class.

Reply:
Moved serialization and format functions to contextHelper.ts. Also added unit tests for the formatter function.

Reply:
Created Serializer class and LogManager class. Also moved helper functions to logHelper.ts. Please take a look when you get a chance @sadym-chromium .